### DISCUSSION TITLE
Is it possible to display both create date and update date for Docusaurs blog?

### DISCUSSION THREAD

**Discussion by abcpeak** [facebook/docusaurus#8892] [Q&A]
I found the blog only display one date besides the reading time.
Now: `20230413 5mins`

How to make blog display both create date and update date?
Such as: `Creat time 20230413, 5mins, Last update on 20230414`

**Comment by slorber** [comment]
Tracked here: https://github.com/facebook/docusaurus/issues/8657

In the meantime, you can do this yourself using custom frontmatter + swizzling blog components
